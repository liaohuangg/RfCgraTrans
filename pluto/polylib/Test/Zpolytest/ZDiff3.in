#Difference of Lattice intersection woth different lattices
M 2
3 3
2 0 0 
2 1 0
0 0 1
3 3
3 0 0
2 1 0
0 0 1
#readDom["{i,j| 1<= i,j<=10}"]
#readDom["{i,j| 1<= i,j<=10}"]
D 2
4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 100
          1 0 -1 100
 4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 100
          1 0 -1 100
F 9



