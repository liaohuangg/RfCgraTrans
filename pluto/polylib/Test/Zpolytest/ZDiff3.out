
ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    1    0    0
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:3  Equations:1  Rays:2  Lines:0
Constraints 3 4
Equality:   [    1    0   -2 ]
Inequality: [    0    1   -3 ]
Inequality: [    0   -1  102 ]
Rays 2 4
Vertex: [    2    3 ]/1
Vertex: [    2  102 ]/1

UNIONED with

ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    2    0    0
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [    2   -1  100 ]
Inequality: [    1    0   -1 ]
Inequality: [   -4    3 -301 ]
Inequality: [   -1    0  100 ]
Rays 4 4
Vertex: [  100  300 ]/1
Vertex: [    1  102 ]/1
Vertex: [    3  305 ]/3
Vertex: [  300  701 ]/3

UNIONED with

ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    6    0    4
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [   -6    1   -5 ]
Inequality: [   -3    0   98 ]
Inequality: [   12   -3  308 ]
Inequality: [    6    0    1 ]
Rays 4 4
Vertex: [   -1   24 ]/6
Vertex: [   98  603 ]/3
Vertex: [   98  700 ]/3
Vertex: [   -1  612 ]/6

UNIONED with

ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    6    0    2
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [   -6    1   -3 ]
Inequality: [   -1    0   33 ]
Inequality: [   12   -3  304 ]
Inequality: [    6    0   -1 ]
Rays 4 4
Vertex: [    1   24 ]/6
Vertex: [   33  201 ]/1
Vertex: [   99  700 ]/3
Vertex: [    1  612 ]/6
